### DISCUSSION TITLE
jekyll compatibility

### DISCUSSION THREAD

**Discussion by avarndell** [sveltia/sveltia-cms#221] [Q&A]
Will you provide jekyll compatiblity in the admin side?

**Comment by kyoshino** [comment]
I have never tried Sveltia CMS with Jekyll, but Sveltia CMS is framework agnostic and you can generate YAML or JSON [data files](https://jekyllrb.com/docs/datafiles/) supported by Jekyll. You can search “Jekyll Netlify CMS” or “Jekyll Decap CMS” to find some tutorials and starter templates:

- https://medium.com/codex/netlify-cms-for-jekyll-site-31f728ac61c7
- https://dev.to/mrkaushal/jekyll-with-netlify-cms-tutorial-54ii
- https://github.com/NickStees/jekyll-cms
- https://github.com/decaporg/jekyll-decap-cms

Once it works for you, [migrating to Sveltia CMS](https://github.com/sveltia/sveltia-cms?tab=readme-ov-file#migration) should be easy!

**Reply by kyoshino** [reply]
Well, Decap CMS actually has the [documentation for Jekyll](https://decapcms.org/docs/jekyll/). Follow this then migrate to Sveltia. Note that Sveltia CMS doesn’t support Git Gateway and Netlify Identity. You need to use the [GitHub backend](https://decapcms.org/docs/github-backend/) + an [OAuth client](https://decapcms.org/docs/external-oauth-clients/) (we provide [our own client](https://github.com/sveltia/sveltia-cms-auth))
